src/api/common/schemas/get-paths.json

```diff
@@ -6,7 +6,23 @@
     "type": "object",
     "properties": {
       "source": {"$ref": "adjustment"},
-      "destination": {"$ref": "adjustment"},
+      "destination": {
+        "type": "object",
```

Comment:
https://github.com/ripple/ripple-lib-extensions/blob/master/transactionparser/test/fixtures/payment-iou-multipath.json

It sends $300 from 3 source trustlines to 3 destination trustlines